docs/style-guides/git/README.md

```diff
@@ -1,199 +1,289 @@
 
-    ```text
-    # Do...
-    Fix Markdown lint errors
-    ```
+## Overview
 
-3.  Do **not** end the subject line with a period.
+Commits should be clearly documented for how they affect consumers of and contributors to this project. To this end, the project adheres to the [Conventional Commits][conventional-commits] specification, which details structured commit messages for clarifying the impact of each commit.
 
-    ```text
-    # Do not...
-    Fix Markdown lint errors.
-    ```
+## Motivation
 
-    ```text
-    # Do...
-    Fix Markdown lint errors
-    ```
+Contributors to this project should describe the changes they make. Detailed descriptions are valuable to various stakeholders, including other contributors, downstream consumers, library authors, individuals needing to deploy and/or integrate the project, and others. Commit authors are best positioned to accurately describe their work and its implications.
 
-4.  Separate the subject line from the body with a blank line.
+The closer to the change one captures information pertaining to a change's context, motivation, and impact, the easier such information will be to write and the more accurate such information will be. Many individuals consume the changes introduced by contributors, from pull request reviewers to library authors to end consumers of this project. The more a contributor can explain a change, the more readily a change can be understood and used, and the more efficient the development process will be.
 
-    ```text
-    # Do not...
-    This is the subject line
-    This is the body body body body body body body body body body body body
-    body body body body body body body body body body body body body body
-    ```
+Commit messages are the closest text to the change itself. The more information-rich a commit message is, the more useful it will be.
 
-    ```text
-    # Do...
-    This is the subject line
+## Specification
 
-    This is the body body body body body body body body body body body body
-    body body body body body body body body body body body body body body
-    ```
+This project adheres to the [Conventional Commits][conventional-commits] specification, with additional project-specific requirements. The project has very precise rules over how Git commit messages must be formatted. This format leads to a commit history which is easier to read and easier to consume as part of automated build processes.
+
+Each commit message consists of three parts: a **header**, a **body** (optional), and a **footer** (optional). Each part **must** be separated from the other parts by a **blank line**.
+
+```text
+<header>
+
+<body>
+
+<footer>
+```
+
+The **header** is **mandatory** and must conform to the [commit message header](#commit-message-header) format.
+
+The **body** is **mandatory** for all non-trivial commits. When present, the body must be at least 20 characters in length and must conform to the [commit message body](#commit-message-body) format.
```

Comment:
I think maybe including an example of a non-trivial commit may clarify what's the severity of the fix and the case that we should be looking to add a body to our commits from now on.

Reply:
Agreed. My plan is to add a list of user stories showing commit examples. Will try to get to later this evening/weekend.

Reply:
I will add user stories in a follow-up PR.